mslib/mswms/mpl_lsec_styles.py

```diff
@@ -0,0 +1,167 @@
+# -*- coding: utf-8 -*-
+"""
+
+    mslib.mswms.mpl_lsec_styles
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Matplotlib linear section styles.
+
+    This file is part of mss.
+
+    :copyright: Copyright 2021 May Baer
+    :copyright: Copyright 2021 by the mss team, see AUTHORS.
+    :license: APACHE-2.0, see LICENSE for details.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+"""
+
+import numpy as np
```

Comment:
Whole file is outdated, I guess?

Reply:
How so? It isn't.

Reply:
Then we might need to talk. I assume this is required for providing the images, and those are not requested by the client anymore?
Even then, I believe a single generic plotting class to be more useful than the specialized ones below.